pkg/users/account-create-dialog.js

```diff
@@ -118,6 +127,23 @@
         return _("The full name must not contain colons.");
 }
 
+function validate_uid(uid, accounts, min_id, max_id, change) {
+    const uid_number = Number(uid);
+    if (!Number.isInteger(uid_number) || uid_number < 0)
+        return _("User ID must be a positive integer");
+
+    if (uid_number < min_id)
+        return cockpit.format(_("User ID must not be lower than $0"), min_id);
+
+    if (uid_number > max_id)
+        return cockpit.format(_("User ID must not be higher than $0"), max_id);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230222-095918-e839713c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)